elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,200 +1,367 @@
 @import "../../pfe-sass/pfe-sass";
 
-/// ===========================================================================
-/// Component Specific SASS Vars
-/// ===========================================================================
+$LOCAL: cta;
+$pfe-cta__arrow-size: 0.313em;
 
-$pfe-cta__arrow-size:                        0.313em;
-
-/// ===========================================================================
-/// DEFAULT CTA
-/// ===========================================================================
 :host {
-  --pfe-cta--main:                     #{pfe-color(ui-link)};
-  --pfe-cta--main--hover:              #{pfe-color(ui-link--hover)};
-  --pfe-cta--main--focus:              #{pfe-color(ui-link--focus)};
-  --pfe-cta--main--visited:            #{pfe-color(ui-link--visited)};
-  --pfe-cta--aux:                      transparent;
-  --pfe-cta--aux--hover:               transparent;
+  --pfe-cta--BorderRadius: 0;
+  --pfe-cta__arrow--Display:  inline;
+  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+
+  --pfe-cta--BackgroundColor:         transparent;
+  --pfe-cta--BackgroundColor--hover:  transparent;
+  --pfe-cta--BackgroundColor--focus:  rgba(0,102,204, .1);
+
+  --pfe-cta--BorderColor:        transparent;
+  --pfe-cta--BorderColor--hover: transparent;
+  --pfe-cta--BorderColor--focus: transparent;
+
+  --pfe-cta--Color:         #{pfe-color(ui-link)};
+  --pfe-cta--Color--hover:  #{pfe-color(ui-link--hover)};
+  --pfe-cta--Color--focus:  #{pfe-color(ui-link--focus)};
+
+  --pfe-cta__inner--BorderColor:        transparent;
+  --pfe-cta__inner--BorderColor--hover: transparent;
+  --pfe-cta__inner--BorderColor--focus: transparent;
+
+  --pfe-cta--Padding: .6rem 0;
 
   display: inline-block;
-  font-weight: bold;
+  font-family: #{pfe-var(font-family--heading)};
+  font-size:   #{pfe-var(font-size)};
+  font-weight: #{pfe-var(font-weight--bold)};
+  line-height: #{pfe-var(line-height)};
 
-  ::slotted(a) {
-    line-height: inherit;
-    color: var(--pfe-cta--main);
-    transition: all #{pfe-var(animation-timing)};
-
-    &::after {
-      display: block;
-      margin-left: calc(#{pfe-var(content-spacer)} * 0.25); // 1/4 of the base `content-spacer`
-      vertical-align: middle;
-      @include pfe-arrow(right, var(--pfe-cta--main), $pfe-cta__arrow-size, $pfe-cta__arrow-size, true);
-    }
-  }
+  background-color: #{pfe-local(BackgroundColor)};
+  border-color:     #{pfe-local(BorderColor)};
 
-  // & ::slotted(a:visited) {
-  //  color: var(--pfe-cta--main--visited);
-  //  &::after {
-  //    @include pfe-arrow-color(var(--pfe-cta--main--visited));
-  //  }
-  // }
+  position: relative;
+  transition: all #{pfe-var(animation-timing)};
 
-  ::slotted(a:hover) {
-    color: var(--pfe-cta--main--hover);
+  line-height: inherit;
```

Comment:
on line 39, we should update `all` and change it to `padding`

Reply:
If we put the padding on the :host element, it works in IE11 but the default padding looks a little bigger.  Do you know why?

Reply:
padding is on host now :)